projects/cps-ui-kit/src/lib/components/cps-file-upload/cps-file-upload.component.ts

```diff
@@ -56,6 +67,18 @@ export class CpsFileUploadComponent implements OnInit, OnChanges {
   @Input() fileProcessingCallback?: (file: File) => Observable<boolean> =
     undefined;
 
+  /**
+   * Position of infoTooltip, it can be 'top', 'bottom', 'left' or 'right'.
+   * @group Props
+   */
+  @Input() infoTooltipPosition: CpsTooltipPosition = 'top';
```

Comment:
`infoTooltip` within the library specifically refers to the tooltip displayed in the `CpsInfoCircleComponent`, which is nested inside other components.
This component doesn't contain info circle component, that's why such name can't be used.